src/styles/blocks/stars.css

```
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('../../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  margin-bottom: 4px;
+}
+
+.stars--1 :first-child {
```

```suggestion
.stars--1 :first-child,
stars--2 :nth-child(-n + 2),
... {
```
Consider using commas to group selectors